hooks/pre-command

```diff
@@ -134,7 +136,8 @@ $(aws ecr get-login --no-include-email)
 default_repository_name="build-cache/${BUILDKITE_ORGANIZATION_SLUG}/${BUILDKITE_PIPELINE_SLUG}"
 repository_name="${BUILDKITE_PLUGIN_DOCKER_ECR_CACHE_ECR_NAME:-${default_repository_name}}"
 max_age_days="${BUILDKITE_PLUGIN_DOCKER_ECR_CACHE_MAX_AGE_DAYS:-30}"
-upsert_ecr "${repository_name}" "${max_age_days}"
+ecr_tags="${BUILDKITE_PLUGIN_DOCKER_ECR_CACHE_ECR_TAGS}"
```

Comment:
Can you please add this to the doco?

I imagine you'd want to support:

```yaml
ecr-tags:
  Key: Value
  Key2: Value2
```

If so you'll need to map these (with bash foo)